Standalone/PythonTests/Automated/benchmark_data_aggregator.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
```

Comment:
This is just moving the changes that were in https://github.com/o3de/o3de/pull/1845/commits/74b995f7e7c386bcca059173aa64acbd6e8398f9, correct?

Reply:
Yes, but note that I modified the script-like execution into a class-based one instead.